fendermint/app/src/cmd/genesis.rs

```diff
@@ -189,9 +191,18 @@ where
 fn into_tendermint(genesis_file: &PathBuf, args: &GenesisIntoTendermintArgs) -> anyhow::Result<()> {
     let genesis = read_genesis(genesis_file)?;
     let genesis_json = serde_json::to_value(&genesis)?;
+
+    // Fendermint chainIDs can only be 50 bytes long. To avoid
+    // arbitrary chain names from exceeding the limit, we compute
+    // the chainID as the 50 most significant bytes of the hex enconding
+    // of the SHA256 of the chain name
+    let mut hasher = sha2::Sha256::new();
+    hasher.update(genesis.chain_name);
+    let h_chain_id = hex::encode(hasher.finalize());
```

Comment:
I thought we agreed that we will use the numeric chain ID itself, which we calculate [here](https://github.com/consensus-shipyard/fendermint/blob/b1dc6c1218331b39553c5990db2f6f8e6513f5ff/fendermint/vm/interpreter/src/fvm/genesis.rs#L84). I think it's preferable because that's what we'll show to the Ethereum API as well. No need to introduce yet another kind of ID for the chain.

Reply:
I think I misunderstood you, then, but your suggestion makes more sense. Nevermind, already fixed.